src/lib/pages/pool/components/PoolRelatedTxs.tsx

```diff
@@ -9,8 +9,24 @@ import {
 } from "@chakra-ui/react";
 
 import { CustomTab } from "lib/components/CustomTab";
+import { useTxsCountByPoolId } from "lib/services/txService";
+
+import { PoolRelatedTxsTable } from "./table/pool-txs";
+
+interface PoolRelatedTxsProps {
+  poolId: number;
+}
+
+export const PoolRelatedTxs = ({ poolId }: PoolRelatedTxsProps) => {
+  const { data: countAllTxs = 0 } = useTxsCountByPoolId(poolId, "is_all");
+  const { data: countSwapTxs = 0 } = useTxsCountByPoolId(poolId, "is_swap");
+  const { data: countLpTxs = 0 } = useTxsCountByPoolId(poolId, "is_lp");
+  const { data: countBondTxs = 0 } = useTxsCountByPoolId(poolId, "is_bond");
+  const { data: countSuperfluidTxs = 0 } = useTxsCountByPoolId(
+    poolId,
+    "is_superfluid"
+  );
 
-export const PoolRelatedTxs = () => {
   return (
     <Box>
       <Flex mt={12} gap={2} alignItems="center">
```

Comment:
create `const tableHeaderId = "poolTableHeader"` and add `id={tableHeaderId}` to heading, and pass it to `PoolRelatedTxsTable` for pagination scroll